docs/pages/api-reference/liveblocks-react-tiptap.mdx

```diff
@@ -80,6 +80,737 @@ Learn more in our [get started guides](/docs/get-started/text-editor/tiptap).
+optionally accepts a shortcut, which is displayed in the tooltip. Comment key
+names are converted to symbols. Here are various examples.
+
+```tsx
+import { Toolbar } from "@liveblocks/react-tiptap";
+import { Icon } from "@liveblocks/react-ui";
+
+// Tooltip: Questio [⌘+Q]
+<Toolbar.Button name="Question" shortcut="CMD+Q" onClick={/* ... */} />
+
+// Custom icon, replaces the name in the button
+<Toolbar.Button name="Question" icon={<div>?</div>} onClick={/* ... */} />
+
+// Using a Liveblocks icon, replaces the name in the button
+<Toolbar.Button name="Question" icon={<Icon.QuestionMark />} onClick={/* ... */} />
+
+// Passing children visually replaces the `name` and `icon`
+<Toolbar.Button name="Question" onClick={/* ... */}>
+  ? Ask a question
+</Toolbar.Button>
+
+// Props are passed to the inner `button`
+<Toolbar.Button
+  name="Question"
+  style={{ marginLeft: 10 }}
+  className="custom-button"
+  onMouseOver={() => console.log("Hovered")}
+/>
+```
+
+##### Props [#ToolbarButton-props]
+
+<PropertiesList>
+  <PropertiesListItem name="name" type="string" required>
+    The name of this button displayed in its tooltip. Will also be displayed in
+    the button if no `icon` or `children` are passed.
+  </PropertiesListItem>
+  <PropertiesListItem name="icon" type="ReactNode">
+    An optional icon displayed in this button.
+  </PropertiesListItem>
+  <PropertiesListItem name="shortcut" type="string">
+    An optional keyboard shortcut displayed in this button’s tooltip. Common
+    shortcuts such will be replaced by their symbols, for example `CMD` → `⌘`.
+  </PropertiesListItem>
+</PropertiesList>
+
+#### Toolbar.Toggle
+
+A toggle button for values that can be active or inactive. Best used with text
+editor commands. The `name` is displayed in a tooltip. Props will be passed to
+the underlying `button` element.
+
+```tsx
+import { Toolbar } from "@liveblocks/react-tiptap";
+
+<Toolbar editor={editor} />
+  <Toolbar.Toggle
+    name="Highlight"
+    active={editor?.isActive("highlight") ?? false}
+    onClick={() => editor?.chain().focus().toggleHighlight().run()}
+  />
+</Toolbar>
+```
+
+The snippet above shows how to use the Toggle with the
+[Tiptap highlight extension](https://tiptap.dev/docs/editor/extensions/marks/highlight).
+The toggle button can also be toggled with `useState`.
+
+```tsx
+import { Toolbar } from "@liveblocks/react-tiptap";
+import { Editor } from "@tiptap/react";
+import { useState } from "react";
+
+function CustomToggle({ editor }: { editor: Editor | null }) {
+  const [active, setActive] = useState(false);
+
+  return (
+    <Toolbar.Toggle
+      name="Toggle options"
+      active={active}
+      onClick={() => setActive(!active)}
+    />
+  );
+}
+```
+
+`Toolbar.Toggle` optionally takes an icon which will `visually` replace the
+`name`. Also optionally accepts a shortcut, which is displayed in the tooltip.
+Comment key names are converted to symbols. Here are various examples.
```

Comment:
`name` is only used in the tooltip, not as a label in the button, so this wouldn't be true, `<Toolbar.Toggle name="Example" />` is an empty button at the moment (same for `Toolbar.Button`)